# HG changeset patch
# User John Donoghue
# Date 1425565471 18000
#      Thu Mar 05 09:24:31 2015 -0500
# Node ID 4e15e8cb16ae73e675eb765ba949c45a57182ba4
# Parent  7d888f9e94eb5fc32e27ec34314fd25779d4a9ac
gui: added icons menu items that have icons used elsewhere

* libgui/src/history-dock-widget.cc
  history_dock_widget::ctxMenu): use icon for copy and create script.

* libgui/src/main-window.cc
  (main_window::construct_edit_menu): use icon on find files menu

diff --git a/libgui/src/history-dock-widget.cc b/libgui/src/history-dock-widget.cc
--- a/libgui/src/history-dock-widget.cc
+++ b/libgui/src/history-dock-widget.cc
@@ -174,20 +174,22 @@ history_dock_widget::update_filter_histo
 
   _filter->insertItem (0, text);    // (re)insert at beginning
   _filter->setCurrentIndex (0);
 }
 
 void history_dock_widget::ctxMenu (const QPoint &xpos)
 {
   QMenu menu (this);
-  menu.addAction (tr ("Copy"), this, SLOT (handle_contextmenu_copy (bool)));
+  menu.addAction (resource_manager::icon ("edit-copy"),
+                  tr ("Copy"), this, SLOT (handle_contextmenu_copy (bool)));
   menu.addAction (tr ("Evaluate"), this,
                   SLOT (handle_contextmenu_evaluate (bool)));
-  menu.addAction (tr ("Create script"), this,
+  menu.addAction (resource_manager::icon ("document-new"),
+                  tr ("Create script"), this,
                   SLOT (handle_contextmenu_create_script (bool)));
   menu.exec (_history_list_view->mapToGlobal (xpos));
 }
 
 void history_dock_widget::handle_contextmenu_copy (bool)
 {
   QString text;
   QItemSelectionModel *selectionModel = _history_list_view->selectionModel ();
diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -1709,17 +1709,19 @@ main_window::construct_edit_menu (QMenuB
     = edit_menu->addAction (tr ("Select All"), this, SLOT (selectAll ()));
 
   _clear_clipboard_action
     = edit_menu->addAction (tr ("Clear Clipboard"), this,
                             SLOT (clear_clipboard ()));
 
   edit_menu->addSeparator ();
 
-  _find_files_action = edit_menu->addAction (tr ("Find Files..."));
+  _find_files_action 
+    = edit_menu->addAction (resource_manager::icon ("edit-find"),
+                             tr ("Find Files..."));
 
   edit_menu->addSeparator ();
 
   _clear_command_window_action
     = edit_menu->addAction (tr ("Clear Command Window"));
 
   _clear_command_history_action
     = edit_menu->addAction (tr ("Clear Command History"));
